flips/20210824-fcl-tx-payer-service.md

```diff
@@ -0,0 +1,331 @@
+# FCL Transaction Payer Service
+
+| Status        | Proposed                                             |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #**    | [612](https://github.com/onflow/flow/pull/612)       |
+| **Author(s)** | Jeffrey Doyle (jeffrey.doyle@dapperlabs.com),        |
+| **Sponsor**   |                                                      |
+| **Updated**   | 2020-09-03                                           |
+
+## Objective
+
+FCL provides a mechanism for a transactions fees to be paid by an account
```

Comment:
@JeffreyDoyle 'FCL provides a mechanism for transactions fees to be paid by an account'?

Reply:
Removed this line!